plugins/catalog-backend-module-gitlab/package.json

```diff
@@ -1,72 +1,76 @@
-  },
-  "typesVersions": {
-    "*": {
-      "alpha": [
-        "src/alpha.ts"
-      ],
-      "package.json": [
-        "package.json"
-      ]
-    }
-  },
   "backstage": {
     "role": "backend-plugin-module"
   },
-  "homepage": "https://backstage.io",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/backstage/backstage",
-    "directory": "plugins/catalog-backend-module-gitlab"
-  },
-  "keywords": [
-    "backstage"
-  ],
-  "scripts": {
-    "start": "backstage-cli package start",
-    "build": "backstage-cli package build",
-    "lint": "backstage-cli package lint",
-    "test": "backstage-cli package test",
-    "prepack": "backstage-cli package prepack",
-    "postpack": "backstage-cli package postpack",
-    "clean": "backstage-cli package clean"
-  },
+  "configSchema": "config.d.ts",
   "dependencies": {
     "@backstage/backend-common": "workspace:^",
     "@backstage/backend-plugin-api": "workspace:^",
     "@backstage/backend-tasks": "workspace:^",
     "@backstage/catalog-model": "workspace:^",
     "@backstage/config": "workspace:^",
     "@backstage/integration": "workspace:^",
+    "@backstage/plugin-catalog-common": "workspace:^",
     "@backstage/plugin-catalog-node": "workspace:^",
+    "@backstage/plugin-events-node": "workspace:^",
+    "@gitbeaker/rest": "^40.0.3",
     "lodash": "^4.17.21",
     "node-fetch": "^2.6.7",
     "uuid": "^9.0.0"
   },
+  "description": "A Backstage catalog backend module that helps integrate towards GitLab",
   "devDependencies": {
     "@backstage/backend-test-utils": "workspace:^",
     "@backstage/cli": "workspace:^",
+    "@backstage/plugin-events-backend-test-utils": "workspace:^",
     "@types/lodash": "^4.14.151",
     "@types/uuid": "^9.0.0",
     "luxon": "^3.0.0",
     "msw": "^1.0.0"
   },
+  "exports": {
+    ".": "./src/index.ts",
+    "./alpha": "./src/alpha.ts",
+    "./package.json": "./package.json"
+  },
   "files": [
     "config.d.ts",
     "dist"
   ],
-  "configSchema": "config.d.ts"
+  "homepage": "https://backstage.io",
+  "keywords": [
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "name": "@backstage/plugin-catalog-backend-module-gitlab",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "directory": "plugins/catalog-backend-module-gitlab",
+    "type": "git",
+    "url": "https://github.com/backstage/backstage"
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint": "backstage-cli package lint",
+    "postpack": "backstage-cli package postpack",
+    "prepack": "backstage-cli package prepack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test"
+  },
+  "types": "src/index.ts",
+  "typesVersions": {
+    "*": {
+      "alpha": [
+        "src/alpha.ts"
+      ],
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "version": "0.3.15-next.1"
```

Comment:
not sure what happened with the field order sorting here :) version is usually way up at the top etc, but we can sort (hah!) that out some other time